src/main.rs

```diff
@@ -121,7 +111,7 @@ CMD [ "tremor", "test", "bench", "/bench", "-o" ]
     fs::write(
         "docker-compose.yml",
         format!(
-            r#"version: '3.3'
+            r#"version: '3.8'
```

Comment:
Version `3.8` isn't widely available yet, since we don't require any of the new features we want to stick with 3.3